storage/storage.go

```diff
@@ -348,3 +395,205 @@ func EncodeName(name string) string {
 	}
 	return result
 }
+
+func SeparateMod(ctx context.Context, body []byte, modID, name string, versionID string, modVersion string) (bool, string) {
```

Comment:
Please convert this into a function that loops over an array consisting of `LinuxServer`, `WindowsServer` and `WindowsNoEditor`, so you don't duplicate the same code 3x.

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/2ac1243aed7bd5d4641f015af08a919157fc46a0
https://github.com/satisfactorymodding/smr-api/pull/12/commits/c078aa8776d72806e5fb249eef0a7d77d31c3929

Let me know if this is up to par... ignore cleanPlatform, it was a foolish idea, and I have removed it.